charts/opentelemetry-collector/values.schema.json

```diff
@@ -2,30 +2,446 @@
   "$schema": "http://json-schema.org/schema#",
   "type": "object",
   "title": "Values",
+  "additionalProperties": false,
   "properties": {
+    "nameOverride": {
+      "description": "Override name of the chart used in Kubernetes object names.",
+      "type": "string"
+    },
+    "fullnameOverride": {
+      "description": "Override fully qualified app name.",
+      "type": "string"
+    },
+    "config": {
+      "description": "Configuration that applies to both standalone and agent collector. Overwritable by standalone and agent specific configs.",
+      "type": "object"
+    },
+    "image": {
+      "description": "Image use in both standalone and agent configs",
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "repository": {
+          "type": "string"
+        },
+        "tag": {
+          "type": "string"
+        },
+        "pullPolicy": {
+          "type": "string",
+          "enum": [
+            "IfNotPresent",
+            "Always",
+            "Never"
+          ]
+        }
+      }
+    },
+    "imagePullSecrets": {
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
```

Comment:
It's not a list of strings in this helm chart. Can be changed later, but currently it's a list of objects like {name: myregistrykey}

Reply:
Updated type to be object.